src/bin/agent.rs

```diff
@@ -30,21 +34,6 @@ struct Arguments {
 
 #[tokio::main]
 async fn main() -> Result<()> {
-    // Initialize a Tracing Subscriber
-    let fmt_builder = tracing_subscriber::fmt()
-        .with_file(false)
-        .with_line_number(true)
-        .with_thread_ids(true)
-        .with_env_filter(tracing_subscriber::EnvFilter::from_default_env())
```

Comment:
I see this one is removed. What does this removal entail?

Reply:
ah this is the ability to allow tracing based on `RUST_LOG` env var, will add back the feature